.github/workflows/codeql.yml

```diff
@@ -0,0 +1,98 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '39 20 * * 1'
```

Comment:
Could you add a comment on what time/day this is so it's easier to read?

Reply:
Sure, done for archival reasons.

```*txt
The cron expression '39 20 * * 1' breaks down as follows:
39 → Minute (Runs at the 39th minute of the hour)
20 → Hour (Runs at 20:39 UTC)
* → Any day of the month
* → Any month
1 → Monday (Runs every Monday at 20:39 UTC)
```